### ISSUE TITLE
Issue: Deprecated jQuery functions on Appraisal Tab

### ISSUE THREAD

**Issue by ross-spencer** [artefactual/archivematica#872]
When a message box is brought up in the appraisal tab, e.g. to relay the successful creation of a SIP when arranging digital objects in ArchivesSpace, a jQuery deprecation warning is raised in the developer window as demonstrated in the graphic below. 

![image](https://user-images.githubusercontent.com/1880412/34546070-171ed81a-f0b7-11e7-8fd9-b3d1d2606c4c.png)

**Comment by ablwr** [comment]
I believe @replaceafill's recent work has fixed this?

**Comment by replaceafill** [comment]
@ablwr No, it's still there. But I just tried this [quick fix](https://gist.github.com/seyDoggy/66978b761fb8572250f7#gistcomment-1644209) to the [AlertDisplayController template](https://github.com/artefactual/archivematica/blob/2ea8ab375bbe2f668ce48622f446bf8b5235f29b/src/dashboard/frontend/app/front_page/appraisal_tab.html#L33) and the warnings are gone.
